### PR TITLE
Travis CI: Build PRs once

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,10 @@ python:
   - "2.7"
 node_js: 
   - "8"
+branches:
+  only:
+    - master
+    - develop
```
Comment (axic) [inline .travis.yml:9]: Though it would make sense renaming  `develop` to `master` before merging this?

Comment (chfast) [reply .travis.yml:9]: It handles both. Later you can remove "develop" but it's not a big deal to keep it there forever.

Comment (winsvega) [reply .travis.yml:9]: will this build two branches?

Comment (pirapira) [reply .travis.yml:9]: no.

**Pull request merged**